pytest_splunk_addon/standard_lib/sample_generation/pytest_splunk_addon_data_parser.py

```diff
@@ -43,6 +45,7 @@ def __init__(self, addon_path: str, config_path: str):
         self._psa_data = None
         self.addon_path = addon_path
         self.match_stanzas = set()
+        self._path_to_samples = self._path_to_samples()
 
     def _path_to_samples(self):
```

Comment:
@uoboda-splunk  ,
I'm reading this function and I'm still not sure what was an intention of self._path_to_samples variable (line 48)

Reply:
The intention was to call this method only once because it will always return the same value